bin/curriculum/backfill-allow-multiple-attempts.rb

```diff
@@ -78,7 +78,8 @@ def backfill_match_levels
     update_dsl_level(level, false)
   end
 
-  Match.all.filter {|l| l.allow_multiple_attempts.nil?}.each do |level|
+  contained_level_names = contained_match_levels.map(&:name)
+  Match.all.filter {|l| l.allow_multiple_attempts.nil? && !contained_level_names.include?(l.name)}.each do |level|
```

Comment:
Sometimes, where `allow_multiple_attempts` was false, it was registering as `nil`. This just more explicitly excludes any levels that should not have the values of `allow_multiple_attempts` changed.